README.md

```diff
@@ -33,59 +33,88 @@ This is useful for datasets that *cannot* pass at the moment due to lack of cove
+|  ds002 | | multiple tasks, multiple runs, in-plane T2, events, T1w, BOLD | https://openneuro.org/datasets/ds000002/versions/00002 |
+|  ds003 | | single task, single run, in-plane T2, events, T1w, BOLD | https://openneuro.org/datasets/ds000003/versions/00001 |
+|  ds005 | | single task, multiple runs, in-plane T2, events, T1w, BOLD | https://openneuro.org/datasets/ds000005/versions/00001 |
+|  ds006 | | single task, multiple sessions, multiple runs, in-plane T2, events, T1w, BOLD | https://openneuro.org/datasets/ds000006/versions/00001 |
+|  ds007 | | single task, multiple runs, in-plane T2, events, T1w, BOLD | https://openneuro.org/datasets/ds000007/versions/00001 |
+|  ds008 | | multiple tasks, multiple runs, in-plane T2, events, T1w, BOLD | https://openneuro.org/datasets/ds000008/versions/00001 |
+|  ds009 | | multiple tasks, multiple runs, in-plane T2, events, T1w, BOLD | https://openneuro.org/datasets/ds000009/versions/00002 |
+|  ds011 | | multiple tasks, multiple runs, T1w, BOLD | https://openneuro.org/datasets/ds000011/versions/00001 |
+|  ds051 | | multiple tasks, multiple runs, T1w, BOLD, inplane T2 | https://openneuro.org/datasets/ds000051/versions/00001 |
+|  ds052 | | multiple tasks, multiple runs, T1w, BOLD, inplane T2 | https://openneuro.org/datasets/ds000052/versions/00001 |
+|  ds101 | | single task, multiple runs, T1w, BOLD | https://openneuro.org/datasets/ds000101/versions/00004 |
+|  ds102 | | single task, multiple runs, T1w, BOLD | https://openneuro.org/datasets/ds000102/versions/00001 |
+|  ds105 | | single task, multiple runs, T1w, BOLD | https://openneuro.org/datasets/ds000105/versions/00001 |
+|  ds107 | | single task, multiple runs, T1w, BOLD | https://openneuro.org/datasets/ds000107/versions/00001 |
+|  ds108 | | single task, multiple runs, T1w, BOLD | https://openneuro.org/datasets/ds000108/versions/00002 |
+|  ds109 | | multiple tasks, multiple runs, T1w, BOLD | https://openneuro.org/datasets/ds000109/versions/00001 |
+|  ds110 | | single task, multiple runs, T1w, BOLD, in plane T2 | https://openneuro.org/datasets/ds000110/versions/00001 |
+|  ds113b | | forrest gump watching, multiple sessions, multiple runs, T1w, T2w, BOLD, angiography, dwi, fieldmaps | https://openneuro.org/datasets/ds000113/versions/1.3.0 |
+|  ds114 | | multiple tasks, multiple runs, T1w, BOLD, DWI | https://openneuro.org/datasets/ds000114/versions/1.0.1 |
+|  ds116 | | multiple tasks, multiple runs, T1w, BOLD, inplane T2 | https://openneuro.org/datasets/ds000116/versions/00003 |
+|  ds210 | | multiple tasks, multiple runs, T1w, BOLD | https://openneuro.org/datasets/ds000210/versions/00002 |
+|  hcp_example_bids | @robertoostenveld |   | https://bit.ly/2H0Z6Qt |
+|  synthetic | @effigies | A synthetic dataset | mri |  |  |  |  | n/a
+|  ds000001-fmriprep | @effigies | Common derivatives example | mri | | | | | https://openneuro.org/datasets/ds000001/versions/1.0.0
+
+## Multimodal datasets
```

Comment:
We could potentially have:

- either one column that list all the modalities rather than having a column header + a repeat of that column header on some given rows, 

|  name | maintained by | description | modalities | link to full data |
| --- | --- | --- | --- | --- | 
|  ds000117 | @RikHenson | A multi-subject, multi-modal human neuroimaging dataset of 19 subjects on a MEG visual task | mri, meg, eeg | https://openneuro.org/datasets/ds000117/ |

- we could have under each modality a list of the dataypes  that one can find I that dataset

|  name | maintained by | description | mri | meg | eeg | ieeg | genetics | link to full data |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  ds000117 | @RikHenson | A multi-subject, multi-modal human neuroimaging dataset of 19 subjects on a MEG visual task | T1w, bold | meg | eeg |  | | https://openneuro.org/datasets/ds000117/ |

Not that urgent. Should not prevent this to be merged. 
Just thinking of what could help users navigate this repo faster to find what they need.

Reply:
that's a good point, given that we'll soon have qMRI, PET, and ASL examples as well ... I like both suggestions ... maybe collapse the columns into one `modalities` and one `datatypes`, and the cell entries are a comma separated list?

Reply:
I am good with that too. :-)